app/controllers/reviews_controller.rb

```diff
@@ -1,7 +1,38 @@
 class ReviewsController < ApplicationController
-  def edit
+  before_action :set_review, only: [:edit]
```

Comment:
please can you add :destroy and :update as well? They will need to be able to access the review as well. other than that, this all looks good, thank you!